### DISCUSSION TITLE
How to improve quality of German TTS

### DISCUSSION THREAD

**Discussion by chm-fp-robotics** [coqui-ai/TTS#1845] [General Q&A]
Hello everyone.

I'm new to Coqui AI and I'm trying to evaluate how to use it with the German TTS model "tts_models/de/thorsten/tacotron2-DCA" (and matching vocoder model "vocoder_models/de/thorsten/fullband-melgan"). Trying simple sentences using the bash commands "tts" or "tts_server" result in a bad audio output quality. Using the default models instead (English) result in a pretty good audio quality. So I'm wondering what I'm doing wrong here.

I saw that config values are being printed on calling these bash commands right after "Setting up Audio Processor..." and these values seem to differ slightly between the default English and the German models. Is there a way to change single parameters to try to achieve better results? And which parameters could affect the bad quality? I'm really a beginner here and I don't know the meaning of these parameters.

**Comment by p0p4k** [comment]
Hello, welcome. The parameters you see are from the pre-trained model. Some of the parameters (like speed) can be changed, but then you have to write the code manually. If you want, you can get your own dataset and fine tune the model by training it.

**Reply by chm-fp-robotics** [reply]
I see, thank you. I just want to use the available voices, not train them myself ;)

**Comment by thorstenMueller** [comment]
Hi @chm-fp-robotics ,
Maybe you can try the newer Thorsten VITS model. It has been trained on a newer dataset and has a better quality than the fullband melgan model.

https://www.thorsten-voice.de/einfach-loslegen/

**Reply by chm-fp-robotics** [reply]
Thank you Thorsten for the quick response and this hint. The VITS model indeed sounds much better quality wise, I'll use this one.

FYI: I basically went through the quick tutorial you showed on your video https://www.youtube.com/watch?v=alpI-DnVlO0&ab_channel=Thorsten-Voice, and there the quality also did not really sound clean, even though the online sound samples of your voice did sound very good.

**Reply by thorstenMueller** [reply]
Hi @chm-fp-robotics ,
just another short update. Since Coqui v0.8 there's my DDC model available which is (in my opinion) even better when it comes to pronounciation.
https://www.thorsten-voice.de/2022/08/22/neues-thorsten-tts-modell-verfuegbar-%f0%9f%a5%b3/

Just in case you want to give this one a try, too.
